includes/class-wp-statistics-option.php

```diff
@@ -76,8 +76,9 @@ public static function defaultOption()
             'attribution_model'               => 'first-touch',
             'geoip_location_detection_method' => 'maxmind',
             'delete_data_on_uninstall'        => false,
-            'share_anonymous_data'             => false,
+            'share_anonymous_data'            => false,
             'display_notifications'           => true,
+            'privacy_audit_issues'            => false,
```

Comment:
Let's use a better name for the option, like
- show_privacy_issues_in_report
- include_privacy_audit_in_email
- show_audit_issues_in_report_email